packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -935,7 +938,12 @@ export class ConnectionManager implements IConnectionManager {
 		// should probably live in the driver.
 		await waitForOnline();
 
-		this.triggerConnect(requestedMode);
+		this.triggerConnect(
+			error !== undefined
+				? "Reconnect on Error"
+				: `Reconnecting due to: ${disconnectMessage}`,
```

Comment:
I would recommend a more consistent format, like maybe:

Reconnecting due to [disconnectMessage] with error [error message]

(that's pseudocode).

Putting the variable stuff in brackets means we could do a simple regex to remove the variable stuff if desired.  And since `reason` and `error` are independent (see other PR), we should always log `reason` (aka `disconnectMessage`).  If you think logging error message will be too random, then just do `with error` and drop the last brackets.

Reply:
If we have both error and reason to reconnect (that other PR), then I'd rather vote to have fixed string here (i.e. no changes) as it will hard to do aggregation (and if needed, we can always add Data_error column to a query).